rails-snow-queen/app/assets/javascripts/quote.js

```diff
@@ -10,40 +10,37 @@ class Quote {
   
   addListeners() {
     $("#submitQuoteModal").on("show.bs.modal", this.handleOpenModal);
-
     const quoteFormNode = document.getElementById("new_quote");
     quoteFormNode.addEventListener("ajax:error", this.handleErrors);
     quoteFormNode.addEventListener("ajax:success", this.handleSuccess);
-
-    $("#successModal").on("hide.bs.modal", () => { location.reload() }) //hacer funcion pa que se vea igual que los otros
-    //abrir el quote modal desde js y no desde html
+    $("#successModal").on("hide.bs.modal", this.handleReload)
   }
 
   handleOpenModal() {
     let quoteData = this.onOpen();
     let polygonsLatLngs = this.getPolygonsJSON(quoteData.polygons);
 
     let splitAddress = quoteData.geocodedAddress.split(",");
-    let addressNodeInModal = document.getElementById("addressModal");
-    addressNodeInModal.innerText = splitAddress;
+    let addressNode = document.getElementById("addressModal");
+    addressNode.innerText = splitAddress;
```

Comment:
Unless you're using any of these variables for anything else, you can tighten this up with just:
```javascript
document.getElementById("addressModal").innerText = splitAddress;
```
In the process saving you the pain of the names (assuming the IDs are descrptive enough)

Reply:
yes